features/ticketing/api/pom.xml

```diff
@@ -8,6 +8,21 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.opennms.features.ticketing</groupId>
   <artifactId>org.opennms.features.ticketing.api</artifactId>
-  <name>OpenNMS Features Ticketing API</name>
-  <url>http://www.opennms.org/</url>
+  <packaging>bundle</packaging>
+  <name>OpenNMS :: Features :: Ticketing :: API</name>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
```

Comment:
This is not really part of this pull request, but I saw this now more often.
We have properties for everything, except this. How about having a JdkVersion8String and a JdkVersion7String or something like that defined in the root pom. Would make migration easier in the future.